repo/incubating/mysql/versions/0/mysql-frameworkversion.yaml

```diff
@@ -0,0 +1,249 @@
+apiVersion: kudo.k8s.io/v1alpha1
+kind: FrameworkVersion
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: mysql-57
+  namespace: default
+spec:
+  serviceSpec:
+  version: "5.7"
+  connectionString: ""
+  framework:
+    name: mysql
+    kind: Framework
+  parameters:
+  - name: BACKUP_FILE
+    description: "Filename to save the backups to"
+    default: "backup.sql"
+    displayName: "BackupFile"
+    trigger: backup
+  - name: PASSWORD
+    default: "password"
+  tasks:
+    deploy:
+      resources:
+      - mysql.yaml
+    init:
+      resources:
+      - init.yaml
+    pv:
+      resources:
+      - backup-pv.yaml
+    backup:
+      resources:
+      - backup.yaml
+    restore:
+      resources:
+      - restore.yaml
+    load-data:
+      resources:
+      - job.yaml
+    clear-data:
+      resources:
+      - job.yaml
+    query:
+      resources:
+      - job.yaml
+  templates:
+    mysql.yaml: |
+      apiVersion: v1
+      kind: Service
+      metadata:
+        name: mysql
+      spec:
+        ports:
+        - port: 3306
+        selector:
+          app: mysql
+        clusterIP: None
+      ---
+      apiVersion: v1
+      kind: PersistentVolumeClaim
+      metadata:
+        name: mysql-pv
+      spec:
+        accessModes:
+          - ReadWriteOnce
+        resources:
+          requests:
+            storage: 1Gi
+      ---
+      apiVersion: apps/v1
+      kind: Deployment
+      metadata:
+        name: mysql
+      spec:
+        selector:
+          matchLabels:
+            app: mysql
+        strategy:
+          type: Recreate
```

Comment:
I am fine with `Recreate` as a strategy but wan't to also think of using other strategies as the preferred way. Is `Recreate` our right choice in this case?